mlflow/tracking/client.py

```diff
@@ -197,6 +386,44 @@ def restore_experiment(self, experiment_id):
         Restore a deleted experiment unless permanently deleted.
 
         :param experiment_id: The experiment ID returned from ``create_experiment``.
+
+        .. code-block:: python
+            :caption: Example
+
+            from mlflow.tracking import MlflowClient
+
+            def print_experiment_info(experiment):
+                print("Name: {}".format(experiment.name))
+                print("Experiment Id: {}".format(experiment.experiment_id))
+                print("Lifecycle_stage: {}".format(experiment.lifecycle_stage))
+
+            # Create and and delete an experiment
+            client = MlflowClient()
+            experiment_id = client.create_experiment("New Experiment")
+            client.delete_experiment(experiment_id)
+
+            # Examine the deleted experiment details. Deleted experiments
+            # are moved to a .trash folder under the artifact URI location top
+            # level directory.
```

Comment:
Is this really correct? `.trash` folder should be created under the root tracking directory (e.g. `mlruns`).

Reply:
Actually, by default without it's stored under ./mlruns. However, upon further inspection, I feel that this is implementation details and it may change. Also, with the SQLAlchemy backend store, we note that info in the `experiment `tables`. 

[sqlite> select *  from experiments;
0|Default|file:///tmp/tracking_dir/0|active
1|New Experiment1|file:///tmp/tracking_dir/1|deleted](url)

Let's remove this detail from the comments.